server/datastore/mysql/activities_test.go

```diff
@@ -415,6 +418,35 @@ func testListHostUpcomingActivities(t *testing.T, ds *Datastore) {
 	sw2Meta, err := ds.GetSoftwareInstallerMetadataByID(ctx, sw2)
 	require.NoError(t, err)
 
+	// insert a VPP app
+	vppCommand1, vppCommand2 := "vpp-command-1", "vpp-command-2"
+	vppApp := &fleet.VPPApp{
+		Name: "vpp_no_team_app_1", VPPAppTeam: fleet.VPPAppTeam{VPPAppID: fleet.VPPAppID{AdamID: "3", Platform: fleet.MacOSPlatform}},
+		BundleIdentifier: "b3",
+	}
+	_, err = ds.InsertVPPAppWithTeam(ctx, vppApp, nil)
+	require.NoError(t, err)
+
+	// install the VPP app on h1
+	commander, _ := createMDMAppleCommanderAndStorage(t, ds)
+	err = ds.InsertHostVPPSoftwareInstall(ctx, h1.ID, vppApp.VPPAppID, vppCommand1, "event-id-1", false)
+	require.NoError(t, err)
```

Comment:
@dantecatalfamo I'm getting a lint error only in the pull request to the RC branch, for a clean cherry-pick. Is there something missing in `minor-fleet-v4.56.0`? ie: something merged in `main` but wasn't cherry-picked?

Reply:
What's the lint error?

Reply:
dang, the github UI only shows it if you go to "Files changed", I thought you'll be able to spot it easily.

> not enough arguments in call to ds.InsertHostVPPSoftwareInstall

Reply:
Possibly this commit? https://github.com/fleetdm/fleet/commit/580a81ded55a017de5028c750271d6c4bf7d2027#diff-964e95bd60bcca3fcec03142ed6a75196e6e75d97465d5383f0363fbb399647cR435

Reply:
ah, interesting, sorry for the wrong ping then! I assumed it was something you fixed!

Reply:
weird that it says "commited 27 days ago"

Reply:
ah no, I think it's 7ed13270930255a1ece9c1386ddffa6db433630e cc: @jahzielv

Reply:
https://github.com/fleetdm/fleet/pull/21629

Reply:
no worries! it happens. Could you approve that PR? 🙏